### ISSUE TITLE
Filed based Turtle + SPARQL example

### ISSUE THREAD

**Issue by chrisgorgo** [linkeddata/rdflib.js#43]
I'm finding it hard to wrap my head around this library. Could you provide me with a simple  example that would:
1. Read a turtle file (from local hard drive)
2. Run a simple SPARQL query on in

Thanks in advance!

**Comment by ckristo** [comment]
Indeed, this library lacks of basic documentation. It took several hours (and a fix for the RDF/XML parser) to read an RDF file from disk.

I need to create some basic examples, when I'm finished I will provide them here...

**Comment by melvincarvalho** [comment]
@chrisfilo this may be of some help ... let me know if you find it readable:

https://github.com/melvincarvalho/clip/blob/master/app/app.js

rdflib doesnt include a sparql engine at present, tho there is one in rdfstore, I tend to use g.any and g.statementsMatching to get the data I need

**Comment by ckristo** [comment]
@chrisfilo as promised, here some examples:
- RDF API example (reading/parsing a file, working with the RDF API, serializing it again): https://gist.github.com/ckristo/9121f31913d21d62c951
- basic SPARQL example: https://gist.github.com/ckristo/2965068ac014b8c87769

**Comment by jeswr** [comment]
Closing as part of the maintainer-signed-off triage (plan #824). This was answered in-thread with working example gists back in 2015, and the README now links API documentation and several tutorials covering parsing local files and querying. If the current docs still leave a gap, please open a fresh issue describing what is missing.
